src/layouts/components/ProjectCard.js

```diff
@@ -1,17 +1,21 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 import styled from 'styled-components'
 
 import { StyledH3, StyledP, StyledUl, StyledLi } from '../../theme/globalStyle'
 import { ButtonSmall } from '../components/Button'
 
+import faker from 'faker'
+
 const Wrapper = styled.div`
-  width: 400px;
+  width: ${props => props.width || 'auto'};
   display: grid;
-  grid-template-columns: repeat(1fr);
-  grid-template-rows: auto;
+  grid-template-columns: 1fr;
+  grid-template-rows: minmax(100px, auto);
```

Comment:
What does this do?

Reply:
it's a css function for the grid. 
here it defines a size range for the row. The row must be greater than or equal to min (first value) and less than or equal to max(last value).

Reply:
Actually, i find it in the video you suggested for **css grid** :smile: